components/Events/Footer.vue

```
@@ -225,8 +225,16 @@ const groupedButtonsByType = computed(
   },
 );
 
-function goToPreScreenigMiddlePage() {
-  navigateTo(`/event/${props.eventSlug}/pre-screening`);
+function goToPreScreeningOrSchedule(eventSlug: string) {
+  // Try to get from localStorage first
+  const lastPreScreening = localStorage.getItem(`lastPreScreening_${eventSlug}`);
```

ESLint

[error] Replace ``lastPreScreening_${eventSlug}`` with `⏎····`lastPreScreening_${eventSlug}`,⏎··`

(prettier/prettier)